CTP/shell/src/com/navercorp/cubridqa/shell/common/CommonUtils.java

```diff
@@ -205,10 +206,26 @@ public static Properties getPropertiesWithPriority(String filename) throws IOExc
 		return props;
 	}
 	
-	public static void writeProperties(String filename,Properties props) throws IOException {
+	public static void writeProperties(String filename,Properties props) {
         File f = new File(filename);
-        OutputStream out = new FileOutputStream( f );
-        props.store(out,"");
+        OutputStream out = null;
```

Comment:
It's better not to catch any exception, just throw them because it is a common method.